test/xpu_api/utilities/optional/optional.object/optional.object.ctor/optional_U_optional_object_ctor.pass.cpp

```diff
@@ -0,0 +1,99 @@
+// -*- C++ -*-
```

Comment:
Consider renaming to something like `rlvalue_optional_U_assign.pass.cpp`

Reply:
Fixed